### DISCUSSION TITLE
Primeng table virtual scroll header jumping while scrolling up - MAC (Firefox)

### DISCUSSION THREAD

**Discussion by ShivrajChougule** [primefaces/community#1207] [PrimeNG] [answered]
**Primeng table & treetable virtual scroll header jumping while scrolling up - Firefox**

![table_virtual_scroll](https://github.com/primefaces/community/assets/36497430/1c023f20-5f70-42ee-a8ae-f103e3fc333b)

The similar issue is mentioned here [#10630](https://github.com/primefaces/primeng/issues/10630)

But, it still exists for the Firefox. PFB links,

https://primeng.org/table#virtual-scroll
https://www.primefaces.org/primeng-v14-lts/table/virtualscroll

**Comment by cetincakiroglu** [comment]
Hi,

I've checked in Firefox developer edition 123.0b6 (64-bit) and Firefox 122.0.1 (64-bit), couldn't replicate the behavior. Which version of firefox do you use?

**Reply by ShivrajChougule** [reply]
Hi @cetincakiroglu,
I'm using 122.0.1 (64-bit) and able to reproduce the issue.

**Reply by cetincakiroglu** [reply]
Is it v14 or v17? If it's v14, we've already fixed the issue. v14 is outdated and will not receive an update. Could you please try it with the newer versions?

**Reply by ShivrajChougule** [reply]
![Screenshot 2024-02-19 at 1 12 06 PM](https://github.com/primefaces/community/assets/36497430/616d0560-e6c4-45f8-8427-c3d71743950d)

I'm using this one, which is already up to date

**Reply by cetincakiroglu** [reply]
I mean the PrimeNG version not firefox.

**Reply by ShivrajChougule** [reply]
primeng version - 17.3.3

https://primeng.org/table#virtual-scroll

**Reply by cetincakiroglu** [reply]
Could you please use the latest version of PrimeNG (17.8.0)

**Reply by ShivrajChougule** [reply]
Tried with 17.9.0, still same issue,
https://primeng.org/table#virtual-scroll

**Reply by ShivrajChougule** [reply]
Hi @cetincakiroglu,
Please not that the issue exist only in MAC firefox.

**Reply by cetincakiroglu** [reply, marked as answer]
Please use the latest version of firefox, we couldn't replicate the issue with the latest versions.
